### PR TITLE
decodeFromString() term entry

### PR DESCRIPTION
**Pull request by christopher-spencer**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/serialization/terms/decodeFromString/decodeFromString.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "name": "Johnny",
+    "age": 30
+  }
+  """
+
+  // Deserialize the JSON string into a Person object using decodeFromString()
+  val person = Json.decodeFromString<Person>(jsonString)
+
+  println(person)
+}
```
Comment (avdhoottt) [inline content/kotlin/concepts/serialization/terms/decodeFromString/decodeFromString.md:50]:
```suggestion
import kotlinx.serialization.*
import kotlinx.serialization.json.Json

@Serializable
data class Person(val name: String, val age: Int)

fun main() {
  val jsonString = """
  {
    "name": "Johnny",
    "age": 30
  }
  """
  // Deserialize the JSON string into a Person object using decodeFromString()
  val person = Json.decodeFromString<Person>(jsonString)
  println(person)
}
```
I think the json class needs to be added separately, its not working in my machine without it. let me know if it's the same for you.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@christopher-spencer Great that you have added the entry in the serialization folder. I have suggested a small change for you to make and the last thing, I think you have mistakenly committed some other file too, please remove that from the PR. :)

**Review by avdhoottt — APPROVED** [review]
Looks Good :)

File: content/kotlin/concepts/serialization/terms/decodeFromString/decodeFromString.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    "age": 30
+  }
+  """
+  // Deserialize the JSON string into a Person object using decodeFromString()
+  val person = Json.decodeFromString<Person>(jsonString)
+  println(person)
+}
+```
+
+This will print the following output:
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/serialization/terms/decodeFromString/decodeFromString.md:52]:
```suggestion
The above example will result in the following output:
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
For now work on these 2 changes @christopher-spencer. I might have more suggestions.

**Review by dakshdeepHERE — APPROVED** [review]
Looks good for second review @christopher-spencer.

**Review by dakshdeepHERE — APPROVED** [review]

**Pull request merged**